en/drafts/originals/analyzing-multilingual-text-nltk-spacy-stanza.md

```diff
@@ -24,91 +24,99 @@ doi: XX.XXXXX/phen0000
+For further reading on these methods, please see the _Programming Historian_ lessons [Sentiment Analysis for Exploratory Data Analysis](https://programminghistorian.org/en/lessons/sentiment-analysis) and [Sentiment Analysis with 'syuzhet' using R](https://programminghistorian.org/en/lessons/sentiment-analysis-syuzhet) for sentiment analysis, and [Finding Places in Text with the World Historical Gazetteer](https://programminghistorian.org/en/lessons/finding-places-world-historical-gazetteer) and [Corpus Analysis with spaCy](https://programminghistorian.org/en/lessons/corpus-analysis-with-spacy) for Named Entity Recognition. The lesson [Introduction to Stylometry with Python](https://programminghistorian.org/en/lessons/introduction-to-stylometry-with-python) may be of interest to those looking to further explore additional applications of computational text analysis.  
 
-Once you have a clean text that is machine-readable, further tasks will be necessary in order to prepare the text for analysis. These are usually referred to as preprocessing tasks. Yet again, however, preprocessing tasks can often involve particular challenges and considerations depending on the types of languages and texts you are working with.
+To prepare the text for computational analysis, we first need to perform certain 'preprocessing' tasks. These tasks can be especially important (and sometimes particularly challenging) when working with multilingual text.
+ 
+For example, you might first need to turn your documents into machine-readable text, using methods such as [Optical Character Recognition (OCR)](https://en.wikipedia.org/wiki/Optical_character_recognition), which can extract text from scanned images. OCR can work very well for many documents, but may give far less accurate results on handwritten scripts, or documents that don't have clearly delineated text (e.g. a document with low contrast between the text and the paper it is printed on). Depending on the languages and texts you work with (and the quality of the OCR method), you might therefore need to 'clean' your text - i.e. correct the errors made by OCR - in order to use it in your analysis. For an introduction to OCR and cleaning, see these _Programming Historian_ lessons: [OCR with Google Vision API and Tesseract](https://programminghistorian.org/en/lessons/ocr-with-google-vision-and-tesseract) and [Cleaning OCR’d text with Regular Expressions](https://programminghistorian.org/en/lessons/cleaning-ocrd-text-with-regular-expressions).
```

Comment:
I’ve removed the reference to [OCR and Machine Translation](https://programminghistorian.org/en/lessons/OCR-and-Machine-Translation), because this lesson is currently not workable, and will be potentially retired in the near future.